pkg/yurtappmanager/webhook/add_yurtappdaemon.go

```diff
@@ -0,0 +1,16 @@
+package webhook
```

Comment:
please add openyurt license at the beginning.

Reply:
done